### PR TITLE
Update copyright notices for 2019 #trivial

### PR DESCRIPTION
**Pull request by djbe**
Same as https://github.com/SwiftGen/SwiftGen/pull/569.

Do note that for consistency & fairness, I've removed people's names from the file headers. They were pretty much obsolete anyway, because multiple of people have worked on the same file, and we keep track of credits using the changelog.

The file headers are now also checked using SwiftLint.

### CODE REVIEW

**Comment by AliSoftware** [comment]
I was gonna suggest to use Danger to check file headers until I saw your comment on the SwiftLint rule, didn't know that one existed nor that it could check a specific pattern 👌

File: README.md [hunk 1]
```diff
@@ -85,3 +85,9 @@ For easier use, you can use the `StencilContext.enrich(context:parameters:enviro
 
 - `param`: the parsed parameters using the parser mentioned above.
 - `env`: a dictionary with all available environment variables (such as `PATH`).
+
+---
+
+# Licence
```
Comment (AliSoftware) [inline README.md]: License/licence we should decide once and for all of we're US or UK 😛

Comment (djbe) [reply README.md]: Most of the existing spelling in the project is the British one, so let's try to keep it.

**Comment by djbe** [comment]
Ugh, yet another bundler issue...

**Comment by AliSoftware** [comment]
Probably just requires a `bundle update` at least for `fuzzy_match`

> Your bundle is locked to fuzzy_match (2.0.4), but that version could not be
found in any of the sources listed in your Gemfile. If you haven't changed
sources, that means the author of fuzzy_match (2.0.4) has removed it. You'll
need to update your bundle to a version other than fuzzy_match (2.0.4) that
hasn't been removed in order to install.

**Comment by AliSoftware** [comment]
Mmh. Even though… [fuzzy_match 2.0.4](https://rubygems.org/gems/fuzzy_match/versions/2.0.4) still seems to exist and haven't been nuked… so maybe just a temporary connection probably to rubygems.org during the `bundle install`…?

**Comment by djbe** [comment]
Yeah, I tried a bundle update locally and everything works fine 😕

**Comment by djbe** [comment]
Seems there was this incident at rubygems:
https://status.rubygems.org/incidents/dh3cd8fjd5qw

Tried logging in via SSH, issue is still present even after deleting `vendor/bundle` and updating `bundler`.

**Pull request merged**